lang/po/Firmware.pot

```diff
@@ -1720,6 +1720,17 @@ msgstr ""
 msgid "THERMAL ANOMALY"
 msgstr ""
 
+#. MSG_TM_CAL c=20 r=4
+#: ../../Firmware/ultralcd.cpp:4054
+msgid "Temp model cal. takes apporx. 12 mins."
```

Comment:
typo: "apporx"

Reply:
looks like a spelling error on the english of all of the translations. not sure if it matters, don't know who's responsibilty it is to fix.